modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -77,7 +77,7 @@ const cellOperations = (mutate: (e1: SugarElement, e2: SugarElement) => void, do
     Css.remove(clone, 'height');
     // dont inherit the width of spanning columns
     if (prev.colspan !== 1) {
-      Css.remove(prev.element, 'width');
```

Comment:
This was pretty hacky and the cause of one of the issues, as it would remove the width on the source cell (this should not of been happening, as that will cause other bugs). The reason it did that was to make sure the cell width wasn't set on the new cell during the `mutate` call. So I've moved that logic to the `CellMutations` logic instead.